### ISSUE TITLE
[A11y]: increase table expand button size

### ISSUE THREAD

**Issue by werdnanoslen** [carbon-design-system/carbon#2737]
## Environment

All environments, this is related to the component itself

## Detailed description

> What version of the Carbon Design System are you using?

Latest v10 and v9, as shown on [the website](https://www.carbondesignsystem.com/components/data-table/code):
![image](https://user-images.githubusercontent.com/764090/57785092-0a1e3280-76ff-11e9-9bba-c11cf888709c.png)

> What did you expect to happen?

Hit box for expand button is small and not as easy as it could be to click.

> What happened instead?

Expand the button (not icon) size to as far up, left, and down as the row's edge and maybe as far right as 8px from the checkbox/content.

> What WCAG 2.1 checkpoint does the issue violate?

2.5.5 Target Size

**Comment by dakahn** [comment]
@carbon-design-system/design 👋🏽

Checked the React version and the click target is the svg icon itself _only_. This gives us a target size of approx 10px. That's a little rough. The WCAG AAA recommendation is 44x44 which is (funnily enough) right around the width of the column that SVG is sitting in.

https://www.w3.org/WAI/WCAG21/Understanding/target-size.html

Since it's a AAA recommendation I ratcheted down the severity and priority, but this is still worth testing and validating/invalidating our assumptions here.

**Comment by shixiedesign** [comment]
@dakahn isn't 44x44 just the **touch target** requirement? Is it now all targets?

**Comment by dakahn** [comment]
No, it's all targets (touch, mouse, etc). But it's **not a requirement**, that's why I lowered the priority and severity. 👍🏽

_"The intent of this success criteria is to ensure that target sizes are large enough for users to easily activate them, even if the user is accessing content on a small handheld touch screen device, has limited dexterity, or has trouble activating small targets for other reasons. For instance, mice and similar pointing devices can be hard to use for these users, and a larger target will help them activate the target."_

It's AAA which is the highest accessibility rating for WCAG compliance though. Something aspirational, but a good guiding light for sure.

That said 10 or 15px is small and extending the padding of the svg/button could prove helpful
![May-15-2019 12-52-27](https://user-images.githubusercontent.com/40970507/57797756-c59b9280-7710-11e9-9729-fb07a3d3b0c6.gif)

**Comment by werdnanoslen** [comment]
Not stale

>
